static/js/giphy.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const renderGrid = require("@giphy/js-components").renderGrid;
+const GiphyFetch = require("@giphy/js-fetch-api").GiphyFetch;
+const throttle = require("throttle-debounce").throttle;
+
+const gf = new GiphyFetch(page_params.giphy_api_key);
+let search_term = "trending";
+
+exports.renderGrid = function (targetEl) {
+    const render = () =>
+        renderGrid(
+            {
+                width: 300,
+                fetchGifs: (offset) =>
+                    gf.search(search_term, {
+                        offset,
+                        limit: 25,
+                        random_id: page_params.user_id,
+                    }),
+                columns: 3,
+                gutter: 6,
+                noLink: true,
+                hideAttribution: true,
```

Comment:
Maybe worth commenting our reasoning for settings like `hideAttribution` as well -- I think `true` is correct here after skimming their docs.

Reply:
okay